experiment_manager:
  ros__parameters:
    # IF A PARAMETER IS SET IN THE LAUNCH FILE, IT SHOULD NOT BE SET HERE SINCE IT WILL BE OVERRIDDEN BY THE LAUNCH FILE
    # The parameters like use_sim_time and use_recorder are placeholders needed to be set in the launch file
    use_sim_time: #THIS IS SET IN THE LAUNCH FILE (default: True) Since the simulation is running in Gazebo, we need to set this to True
    use_recorder: #THIS IS SET IN THE LAUNCH FILE (default: False) (if True, the experiment manager will record the bag files)
    algorithm_name: "test_algorithm"
    base_path: "/workspaces/hunavsim_ws/bags/gazebo_test"
    timeout_duration: 60.0
    repetitions: 1
    goals_and_poses_file: "/workspaces/hunavsim_ws/src/gazebo_test/gazebo_test/goals_and_poses/social_nav.yaml"
      